libs/core/src/lib/api/billing.service.ts

```diff
@@ -90,6 +92,19 @@ export class BillingService {
     );
   }
 
+  getStripeSubscription(): Observable<StripeAccountSubscription | null> {
+    return this.getSubscription().pipe(
+      map((subscription) => {
+        if (!(subscription as unknown as any).provider) {
+          return null;
```

Comment:
you should not return null in this case, is there is no provider it's always a `StripeSubscription`
```suggestion
          return subscription;
```